gammapy/maps/tests/test_wcsnd.py

```diff
@@ -378,3 +379,15 @@ def test_coadd_unit():
 
     assert_allclose(m1.data, 1.0001)
 
+def test_make_region_mask():
+    from regions import CircleSkyRegion
+    geom = WcsGeom.create(npix=(3,3), binsz=2,
+                          proj='CAR', coordsys='GAL')
+    m = WcsNDMap(geom)
+    region = CircleSkyRegion(SkyCoord(0, 0, unit='deg', frame='galactic'), 1.0*u.deg)
+    maskmap = m.make_region_mask(region)
+    assert np.sum(maskmap.data) == 1
```

Comment:
Suggest to add an assert on the dtype of data. It's bool, no?

Reply:
Done